src/main/java/biblioteka/Calendar.java

```diff
@@ -0,0 +1,354 @@
+package biblioteka;
+import model.Thing;
+import model.User;
+import java.sql.*;
+import java.util.List;
+//https://www.postgresqltutorial.com/postgresql-jdbc/ - fajny tutorial, sa tam tez metody do update'u czy kasowania ale narazie niepotrzebne
```

Comment:
It's okay to have such comments in local code, but they should make it to review

Reply:
I delet this